### PR TITLE
Tutorial.rst: Explaination of .coafile

### LINKED ISSUE #4787: Explain the functionality of .coafile in 'Getting Started with coala'
I propose that maybe we should clarify what the .coafile is for in the guide [Getting Started with coala](https://docs.coala.io/en/latest/Users/Tutorial.html).

I am a new user of coala. I am not sure if others agree with me but I think there is one point in the 'Getting Started with coala' guide that is confusing. In the section 'Configuration Files - coafiles', the guide does not really introduce what functionality the .coafile has. It says that it is a configuration file. But what I couldn't find there is that the .coafile is something that allows you to run coala the way you want without specifying all the arguments again. Like it says in the '[about](https://coala.io/#/about)' page of the homepage, it gives one of the 2 options to run coala. I was really confused about why we need the .coafile when I was reading the guide. I have also looked into the [Configuration File](https://docs.coala.io/en/latest/Users/coafile.html) page in the doc to try to find out what it is. It wasn't until I stumbled upon the about section on the homepage that I see what the .coafile is for.

I have not used any tool similar to coala, so maybe this is more obvious to other people.

### PR DESCRIPTION
**Pull request by Kanav-7**
This explains working of .coafile in getting started document

Closes https://github.com/coala/coala/issues/4787

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Review by acashmkj — APPROVED** [review]
LGTM

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -153,6 +153,19 @@ This should yield something like this:
     files = src/*.c
     use_spaces = yeah
 
+.. note::
+
+    - The ``bears key`` specifies which bears (plugins) you want to use.
```
Comment (jayvdb) [inline Users/Tutorial.rst]: "``bears key``" is not a thing.  It is "``bears`` key"

**Review by jayvdb — CHANGES_REQUESTED** [review]

**Comment by Kanav-7** [comment]
@jayvdb Made required changes. Please review.

**Comment by PlentyMore** [comment]
please squash the commits

**Review by prashant0598 — CHANGES_REQUESTED** [review]
@Kanav-7 commit according to commit guidelines  https://coala.io/commit and squash your commits For squashing Refer: https://api.coala.io/en/latest/Developers/Git_Basics.html

**Review by IpshitaC — CHANGES_REQUESTED** [review]
Hey @Kanav-7 
As explained [here](http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-7-review-process), when someone requests a change on your PR, you need to amend the previous commit and not create a new one.
Can you please squash these two commits into a single commit? Refer to [this](http://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits) for more details.

**Comment by Kanav-7** [comment]
@jayvdb @IpshitaC Squashed Commits. Please review (sorry for being late)

**Review by Mohit17067 — APPROVED** [review]
ack a05c5f9df68351620b8a8a33a4c18b29454c4988

**Review by jayvdb — CHANGES_REQUESTED** [review]
The last line of the commit message should contain a URL to the issue.

**Pull request closed without merging**